tests/phpunit/CRM/CiviAwards/BAO/AwardReviewPanelTest.php

```diff
@@ -216,12 +254,37 @@ public function testCreateThrowsExceptionWhenRestrictedStatusIsEmptyAndApplicati
     AwardReviewPanelFabricator::fabricate($params);
   }
 
+  /**
+   * Test create function fails if restricted status is empty.
+   */
+  public function testCreateThrowsExceptionWhenRestrictedStatusIsEmptyAndApplicationStatusIsRestricted() {
```

Comment:
Will be a good idea to separate syntax fixes commit from other code commits next time so it's easier to review the PR.

Reply:
sure